src/aviary/functional.py

```diff
@@ -80,7 +80,7 @@ async def step(
             concurrency=self.allow_concurrency,
             handle_tool_exc=True,
         )
-        return msgs, self.state.reward, self.state.done, False
+        return cast(Messages, msgs), self.state.reward, self.state.done, False
```

Comment:
I want to fix these type errors in another PR, can we use `type: ignore` for now over `cast`